cli/command/stack/kubernetes/deploy.go

```diff
@@ -16,8 +16,8 @@ import (
 func RunDeploy(dockerCli *KubeCli, opts options.Deploy) error {
 	cmdOut := dockerCli.Out()
 	// Check arguments
-	if opts.Composefile == "" {
-		return errors.Errorf("Please specify a Compose file (with --compose-file).")
+	if len(opts.Composefiles) == 0 {
+		return errors.Errorf("Please specify only one compose file (with --compose-file).")
```

Comment:
This feature doesn't work with kubernetes deploy?

Reply:
@dnephin not right now because we parse the file differently. I want to tackle this in a follow-up (i.e. use the same loading/parsing code for both swarm and k8s and thus remove this limitation).